proto/eth/v1alpha1/slasher.proto

```diff
@@ -51,13 +51,15 @@ message ProposerSlashingResponse {
 // each attestation source to those spans
 // see https://github.com/protolambda/eth2-surround/blob/master/README.md#min-max-surround
 // for further details.
-message MinMaxSpan {
-    uint32 min_span = 1;
-    uint32 max_span = 2;
+message MinMaxEpochSpan {
+    uint32 min_epoch_span = 1;
```

Comment:
whats the reason for `uint32` instead of  `uint64` ?

Reply:
less space. i would be good with 16

Reply:
Why not use 16?

Reply:
sadly protobuf dont have uint16